docker-for-windows/troubleshoot.md

```diff
@@ -464,19 +447,12 @@ For more information, see:
 * [Windows NAT capabilities and
   limitations](https://blogs.technet.microsoft.com/virtualization/2016/05/25/windows-nat-winnat-capabilities-and-limitations/)
 
+### Running Docker Desktop in nested virtualization scenarios
 
-### Running Docker Desktop for Windows in nested virtualization scenarios
-
-Docker Desktop for Windows can run inside a Windows 10 virtual machine (VM) running on
-apps like Parallels or VMware Fusion on a Mac provided that the VM is properly
-configured. However, problems and intermittent failures may still occur due to
-the way these apps virtualize the hardware. For these reasons, _**Docker Desktop for
-Windows is not supported for nested virtualization scenarios**_. It might work
+Docker Desktop can run inside a Windows 10 virtual machine (VM) running on apps like Parallels or VMware Fusion on a Mac provided that the VM is properly configured. However, problems and intermittent failures may still occur due to the way these apps virtualize the hardware. For these reasons, _**Docker Desktop is not supported in nested virtualization scenarios**_. It might work
```

Comment:
I think VM is already spelled out, but you many want to check.

Reply:
There were a couple of instances of 'VM' in the previous sections but wasn't spelled out in the first instance. I've fixed that and updated this instance to VM.

Reply:
Fixed in commit 59aa7e5.